### DISCUSSION TITLE
IIP-59 System Deposit

### DISCUSSION THREAD

**Discussion by mksong76** [icon-project/IIPs#65] [General]
Please add any opinions for IIP-59 ICON System Deposit

**Comment by CyrusVorwald** [comment]
I am wondering whether resolving this as a precompile was considered and if so, why this route was selected instead.

**Reply by mksong76** [reply]
If you mean implementing BMV in native for the pre-compiled contract, it breaks our first target, expanding the network without a hard fork(node changes). Otherwise, please describe it in detail.

**Reply by CyrusVorwald** [reply]
That's what I was wondering about, thanks!
